rest/bulk_api.go

```diff
@@ -397,21 +393,35 @@ func (h *handler) handleBulkGet() error {
 				err = base.HTTPErrorf(http.StatusBadRequest, "Invalid doc/rev ID in _bulk_get")
 			} else {
 				attsSince, err = db.GetStringArrayProperty(doc, "atts_since")
-				if revsLimit > 0 {
-					revsFrom, err = db.GetStringArrayProperty(doc, "revs_from")
-					if revsFrom == nil {
-						revsFrom = attsSince // revs_from defaults to same value as atts_since
+
+				if h.getBoolQuery("revs") {
```

Comment:
It would be better to read the query level properties ("revs", "revs_limit") once prior to ranging over docs, instead of re-parsing from the query string in each iteration.